Documentation/sourcebuild-in-repos/new-dependencies.md

```diff
@@ -0,0 +1,46 @@
+1. Are you already using a package from the same repo and is the new dependency already source-built?  You can determine this by checking if the repo has a SourceBuild leg building, e.g. runtime's is `runtime-dev-innerloop (Build Linux x64 release SourceBuild)`.
+	1. This is the simplest case.  You can add new dependencies from repos you are already using freely.  Note that you need to use the same version for all packages from the repo.
+2. Is the repo already built in source-build including the specific package you want?
+	1. Add the dependency using `darc add-dependency` as normal, then add the [source-build metadata](#Basics) as above.
+3. Is the repo already built in source-build but the specific package is not?
+	1. There's probably an issue with source-building this package.  Please talk to a [source-build team member](https://github.com/orgs/dotnet/teams/source-build-internal) about why that is and whether we can fix it.
+4. Is this a Microsoft repo that uses Arcade to build?
+	1. Does the foreign repo depend on your repo, directly or indirectly?  i.e. would adding the dependency create a cycle?
+		1. This isn't necessarily a deal-breaker - it can sometimes be worked around with reference-only packages.  Please contact a [source-build team member](https://github.com/orgs/dotnet/teams/source-build-internal) to discuss.
+	2. Does the foreign repo publish to BAR?
+		1. If not, please contact them to get them publishing to BAR in an appropriate channel.
+	3. If neither of these caveats apply you should be in good shape.  Follow the instructions under "Basics" above.
+5. Source-build has in the past used Arcade shims to allow non-Arcade repos to build appropriate packages for source-build.  Please contact the [source-build team](https://github.com/orgs/dotnet/teams/source-build-internal) to determine if this is a workable approach for your foreign repo.
+
+## External dependencies
+
+External (non-Microsoft or even non-dotnet) dependencies need to be carefully considered for source-build in addition to the licensing issues you are probably already familiar with.
+1. Dependencies that have no code (e.g. SDKs with just props and targets) can usually be added using the [source-build text-only-package process](https://github.com/dotnet/source-build-reference-packages/tree/main/src/textOnlyPackages/src).  If the package is not already included there, you can [open a PR](https://github.com/dotnet/source-build-reference-packages/pulls) or [file an issue](https://github.com/dotnet/source-build/issues/new/choose) to include it.
+2. We build some external dependencies in the [dotnet/source-build-externals](https://github.com/dotnet/source-build-externals) repo.  Good targets for this generally have very few if any dependencies and very simple build processes.  [Contact a source-build team member](https://github.com/orgs/dotnet/teams/source-build-internal) for more information.
```

Comment:
I think this should reference "file an issue" vs contacting the source-build team.  In general we should prefer communicating over issues that than direct pings.

Reply:
Changed as suggested.